ui/app/AppLayouts/Wallet/services/dapps/WalletConnectService.qml

```diff
@@ -30,6 +32,17 @@ QObject {
             value: Constants.watchWalletType
             inverted: true
         }
+        proxyRoles: [
+            FastExpressionRole {
+                name: "colorizedChainPrefixes"
+                function getChainShortNames(chainIds) {
```

Comment:
Worth extracting this into e.g. WalletUtils.qml; it's duplicated all over the place

Reply:
Same for `getNetworkShortNames()`

Reply:
(doesn't have to be in this PR :)

Reply:
It's good idea to exclude, but not to any singleton utils because this method depends on backend's data.

Reply:
Could be a deeper discussion and it would deserve its own space. With the current Store arrangements it would mean to duplicate it in several stores instead of having it here. Moreover, it would mean to add it to some singleton stores as well creating singleton cyclic dependencies.. I'd leave it as it is for now.
IMO adding a task to unify `colorizedChainPrefixes` implementations and remove the singleton access to nim module from WalletUtils would be more appropriate. WDYT?